frontend/src/actions/taskActions.js

```diff
@@ -0,0 +1,55 @@
+import axios from '../plugins/axios';
```

Comment:
I think you can import this in main file (App.js I think?) and then everywhere else you could just do `import axios from 'axios'`

Reply:
Tried and it is now working this way

Reply:
cause you create anonther instance of axios there

in plugins you can do
```
import axios from 'axios';

axios.defaults.baseURL = apiUrl;
axios.defaults.withCredentials = true;

export default axios;
```